packages/kbn-optimizer/limits.yml

```diff
@@ -24,6 +24,7 @@ pageLoadAssetSize:
   esUiShared: 326654
   expressions: 224136
   features: 21723
+  fieldFormats: 120000
```

Comment:
The plugin is currently coming in at 75kb, why put so much leeway here?

Reply:
Thanks! will update. I added this before I improved bundle size in https://github.com/elastic/kibana/pull/107278 :)

Reply:
I recommend deleting this line and running 

```
node scripts/build_kibana_platform_plugins --filter fieldFormats --update-limits
```

Reply:
Thanks, I updated using suggested